src/components/results/doctorInfoList.js

```diff
@@ -1,16 +1,16 @@
 import React from 'react';
 import DoctorInfoListItem from './DoctorInfoListItem.js'
 
-class DoctorInfoList extends React.Component {
+const DoctorInfoList = ({doctorsList}) => {
+
+  return (
+    <ul className="doctor-list">
+        { doctorsList.map((item, index) => <DoctorInfoListItem key={index} {...item} />) }
```

Comment:
[comment] Just want to mention this. React specifically calls out not to use array indexes as keys: https://reactjs.org/docs/lists-and-keys.html. However, it is acceptable when you have no other option (which is the case here). Just keep this in mind on real projects -- if you have a real ID, that is always preferred over index.